ychat/src/commonTest/kotlin/infrastructure/MockStorage.kt

```diff
@@ -29,4 +29,12 @@ object MockStorage {
             "\"allow_logprobs\":true,\"allow_search_indices\":false,\"allow_view\":true," +
             "\"allow_fine_tuning\":false,\"organization\":\"*\",\"group\":null,\"is_blocking" +
             "\":false}],\"root\":\"$id\",\"parent\":null}]}"
+
+    fun modelSuccessResult(id: String) = "{\"id\":\"$id\",\"object\":\"model\"," +
```

Comment:
Multiline String Literals could be  used here for better readability

Reply:
I didn't know that. Indeed increase readability. Adjust made!